core/app/views/refinery/_header.html.erb

```diff
@@ -4,7 +4,7 @@
   <% end %>
 <% end %>
 
-<% cache [I18n.locale, 'main_menu', @page, Refinery::Page.fast_menu.map(&:updated_at).max] do %>
+<% cache [I18n.locale, current_refinery_user, @page, Refinery::Page.fast_menu] do %>
```

Comment:
huh? There is no `current_refinery_user` for this frontend view

Reply:
Yes there is a `current_refinery_user`: https://github.com/refinery/refinerycms/blob/2c3b7c31314cc3fbcd425d7c461e14002bfda7fc/core/lib/refinery/application_controller.rb#L51

Reply:
but it's going to be nil when you're logged out so your cache key will be `en__#{@page}_#{array of pages}` which seems like a strange key. What am I missing?

Reply:
Probably, I will change the way i generate the cache_key, i will also using the model `cache_key` method. But right now i've a `can't dump File` error in the specs and i don't know why.